tests/fips/openssl/openssl_fips_cipher.pm

```diff
@@ -36,7 +36,8 @@ sub run {
     assert_script_run "mkdir fips-test && cd fips-test && echo Hello > $file_raw";
 
     # With FIPS approved Cipher algorithms, openssl should work
-    my @approved_cipher = ("aes128", "aes192", "aes256", "des3", "des-ede3");
+    my @approved_cipher = ("aes128", "aes192", "aes256");
+    push @approved_cipher, qw(des3 des-ede3) unless is_sle('15-SP6+');
```

Comment:
and any product 'newer' - like Tumbleweed :)

Reply:
https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/19325